purchasing/static/less/opportunities/_styles.less

```diff
@@ -1,3 +1,12 @@
+
+a {
```

Comment:
Do you mean to change links globally? This will change all of them across all of Beacon.